crates/matrix-sdk-base/src/client.rs

```diff
@@ -167,7 +167,7 @@ impl BaseClient {
 
     /// Lookup the Room for the given RoomId, or create one, if it didn't exist
     /// yet in the store
-    pub async fn get_or_create_room(&self, room_id: &RoomId, room_type: RoomType) -> Room {
+    pub async fn get_or_create_room(&self, room_id: &RoomId, room_type: RoomState) -> Room {
```

Comment:
Don't we want to rename the variable in this case as well?

```suggestion
    pub async fn get_or_create_room(&self, room_id: &RoomId, room_state: RoomState) -> Room {
```